samples/extensionSamples/yarn.lock

```diff
@@ -1381,7 +1381,7 @@ gulplog@^1.0.0:
   dependencies:
     glogg "^1.0.0"
 
-handlebars@^4.5.3:
+handlebars@^4.7.7:
```

Comment:
So this is already pulling the 4.7.7 version (line 1385)

It looks like the issue here is that the governance alert is coming from stale pipelines which aren't getting updated. Looking at the PR validation pipeline for some reason the component governance isn't being ran for that pipeline (and other ones) :

Product build pipeline : 

![image](https://user-images.githubusercontent.com/28519865/124033733-2c429f80-d9af-11eb-9dad-78d2771b1a19.png)

PR Validation pipeline : 
![image](https://user-images.githubusercontent.com/28519865/124033661-159c4880-d9af-11eb-91ef-6f2a91120df1.png)

So two possible fixes here I see

1. Untrack stale pipelines (where governance isn't running)
2. Get governance running for those pipelines that are still running. Easiest for this would be to manually add the task to the pipeline yaml